nav/nav_stack/config/global_costmap_params.yaml

```diff
@@ -6,8 +6,41 @@ global_costmap:
   static_map: false     # Not supplying a static map
 
   # Full width and height of IGVC map
-  width: 50
-  height: 50
-  origin_x: -50
-  origin_y: -50
+  width: 200 # Arbitrary large values used for now because I'm not sure if 50 x 50 is actually large enough
```

Comment:
@1999michael to check this out

Reply:
Was there a reason you expanded the width/height?

Reply:
When setting the size of the local cost map, the ZED camera only contributes to the cost map when increasing the map size. I played around with the numbers briefly and noticed that when the local size is set to 15 (camera range), it still in large enough to contribute to the cost map. I set it to local size to 40 for now and it seemed to be big enough (maybe a little too big). I wasn't sure whether or not the scaling was correct for both the local and global cost map size, which is why I increased the global cost map in hopes of fixing the issues related to the global cost map (to which it didn't).

Reply:
I feel like it would be best to change it back to the old size for now and correct it if we need to in another issue.

Reply:
Resolved by reverting global cost map size to original size.